--- a/minerl/Malmo/Minecraft/gradle/wrapper/gradle-wrapper.properties
+++ b/minerl/Malmo/Minecraft/gradle/wrapper/gradle-wrapper.properties
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-all.zip
+distributionUrl=https\://mirrors.aliyun.com/gradle/gradle-4.10.2-all.zip
--- a/minerl/Malmo/Minecraft/build.gradle
+++ b/minerl/Malmo/Minecraft/build.gradle
@@ -1,7 +1,10 @@
 // For those who want the bleeding edge
 buildscript {
     repositories {
-
+        maven { url "https://maven.aliyun.com/repository/public" }
+        maven { url "https://maven.aliyun.com/repository/central" }
+        maven { url "https://libraries.minecraft.net/" }
+        maven { url "https://jitpack.io" }
         jcenter()
         mavenCentral()
     }
@@ -306,7 +309,7 @@
     description 'Generate source files for our XML schemas using JAXB'
 
     // Create an index file listing all the schemas:
-    def schemaIndexFile = new File('src/main/resources/schemas.index')
+    def schemaIndexFile = new File(projectDir, 'src/main/resources/schemas.index')
     def contents = ""
     def tree = copySchemas.source
     tree.visit { fileDetails ->

